enterprise/backend/src/metabase_enterprise/ai_sql_fixer/api.clj

```diff
@@ -0,0 +1,38 @@
+(ns metabase-enterprise.ai-sql-fixer.api
+  "`/api/ee/ai-sql-fixer/` routes"
+  (:require
+   [metabase-enterprise.metabot-v3.client :as metabot-v3.client]
+   [metabase.api.macros :as api.macros]
+   [metabase.api.routes.common :refer [+auth]]
+   [metabase.driver :as driver]
+   [metabase.driver.util :as driver.u]
+   [metabase.models.persisted-info :as persisted-info]
+   [metabase.query-processor.compile :as qp.compile]
+   [metabase.query-processor.middleware.permissions :as qp.perms]
+   [metabase.util.malli.schema :as ms]))
+
+(api.macros/defendpoint :post "/fix"
+  "Suggest fixes for a SQL query."
+  [_route-params
+   _query-params
+   {:keys [query error_message]} :- [:map
+                                     [:query [:map
+                                              [:database ms/PositiveInt]
+                                              [:pretty   {:default true} [:maybe :boolean]]]]
```

Comment:
`pretty` should be removed as otherwise line numbers would not match.

Reply:
So the query shown to user will never be generated with `pretty`? Or are they _always_ generated with `pretty`?

Reply:
always without

Reply:
@metamben

Reply:
Done.